### PR TITLE
Remove __main__

### PR DESCRIPTION
**Pull request by gramalingam**
The `__main__.py` file is very old and outdated. Removing it. (If anyone wants this, it should probably be updated/redone.)

### CODE REVIEW

**Review by justinchuby — APPROVED** [review]
That’s a strange failure

**Comment by justinchuby** [comment]
Ah, there's also `cli_test` that needs to be removed

File: onnxscript/__init__.py [hunk 1]
```diff
@@ -63,6 +63,12 @@
     # package is not installed
     pass
 
+def const (pyvalue: Any) -> Any:
```
Comment (justinchuby) [inline onnxscript/__init__.py:66]: typing: T -> T?

File: onnxscript/tests/cli_test.py [hunk 2]
```diff
@@ -7,15 +7,10 @@
 
 import click.testing
 
-from onnxscript.__main__ import onnx2script, translate
+from onnxscript.__main__ import translate
```
Comment (justinchuby) [inline onnxscript/tests/cli_test.py:10]: So there’s still translate? Just curious where it was

File: onnxscript/__init__.py (hunk 1 above)
Comment (gramalingam) [reply onnxscript/__init__.py:66]: Sorry, this crept in accidentally into this PR. It was (meant to be) part of a different branch for an in-progress PR. Will remove it.

File: onnxscript/tests/cli_test.py (hunk 2 above)
Comment (gramalingam) [reply onnxscript/tests/cli_test.py]: That was incomplete, sorry. Removed it entirely.

**Comment by justinchuby** [comment]
I suppose we no longer need click as a requirement?

**Comment by justinchuby** [comment]
https://github.com/microsoft/onnx-script/pull/593

**Pull request merged**